README.md

```diff
@@ -3,148 +3,88 @@
-  to proxy HTTP traffics, or you can configure `cloud_service.yaml` to use [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport)
-  - if using ingress, configure `spec.rules[0].host` to your domain name
-- Deploy mysql on Kubernetes first if you don't have it on your cluster, and modify the mysql endpoint in settings.py
-  - `kubectl create -f ./mysql_deployment.yaml` (you need to fill in the nodeselector field with your node's hostname or ip in yaml first)
-  - `kubectl create -f ./mysql_service.yaml`
-- Deploy cloud on Kubernetes
-  - `kubectl create -f k8s/cloud_deployment.yaml`(you need to fill in the nodeselector field with your node's hostname or ip in yaml first)
-  - `kubectl create -f k8s/cloud_service.yaml`
-  - `kubectl create -f k8s/cloud_ingress.yaml`(optianal if you don't need Jupyter notebook)
-
-
-To test or visit the website, find out the kubernetes ingress IP
-addresses, or the NodePort.
-
-Then open your browser and visit `http://<ingress-ip-address>`, or
-`http://<any-node-ip-address>:<NodePort>`
-
-- Prepare public dataset
-
-  You can create a Kubernetes Job for preparing the public cloud dataset with RecordIO files. You should modify the YAML file as your environment:
-  - `<DATACENTER>`, Your cluster datacenter 
-  - `<MONITOR_ADDR>`, Ceph monitor address
-  ```bash
-  kubectl create -f k8s/prepare_dataset.yaml
-  ```
-
-### Run locally without docker
-
-- You still need a kubernetes cluster when try running locally.
-- Make sure you have `Python > 2.7.10` installed.
-- Python needs to support `OPENSSL 1.2`. To check it out, simply run:
-    ```python
-       >>> import ssl
-       >>> ssl.OPENSSL_VERSION
-       'OpenSSL 1.0.2k  26 Jan 2017'
-    ```
-- Make sure you are using a virtual environment of some sort (e.g. `virtualenv` or
-`pyenv`).
+## Build
 
-```
-virtualenv paddlecloudenv
-# enable the virtualenv
-source paddlecloudenv/bin/activate
-```
+[Build all components and Docker images](./doc/build/build.md)
 
-To run for the first time, you need to:
+## Deploy
 
-```
-cd paddlecloud
-npm install
-pip install -r requirements.txt
-./manage.py migrate
-./manage.py loaddata sites
-npm run dev
-```
+[Deploy Cloud Server](./doc/deploy/deploy.md)
 
-Browse to http://localhost:8000/
+[Deploy EDL](./doc/deploy/deploy_edl.md)
 
-If you are starting the server for the second time, just run:
-```
-./manage.py runserver
-```
+[Run on Minikube (for developers)](./doc/deploy/run_on_minikube.md)
 
-### Configure Email Sending
-If you want to use `mail` command to send confirmation emails, change the below settings:
+## Contribute
+
+We appreciate your contributions! You can contribute on any of the components
+according to the following code structure:
 
 ```
-EMAIL_BACKEND = 'django_sendmail_backend.backends.EmailBackend'
+.
+├── demo: distributed version of https://github.com/PaddlePaddle/book programs
+├── doc: documents
```

Comment:
I think there are two sub dir in doc, `design` and `deploy`

Reply:
It's quite easy to understand what they are under the `doc` directory, so I omitted them.